.github/workflows/yarn-upgrade.yml

```
@@ -0,0 +1,62 @@
+name: Yarn Upgrade
+
+on:
+  schedule:
+    # Every wednesday at 13:37 UTC
+    - cron: 37 13 * * 3
+  workflow_dispatch: {}
+
+jobs:
+  upgrade:
+    name: Yarn Upgrade
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check Out
+        uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@b2.1.0
+        with:
+          node-version: 10
+
+      - name: Locate Yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Restore Yarn cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |-
```

I don't see a reference to this trailing dash anywhere in the docs or other examples. yml magic I don't know about, or typo? Should this be:
```suggestion
          restore-keys: |
```

---

The `-` is the [block chomping indicator](https://yaml.org/spec/1.2/spec.html#id2794534) for stripping the final break and any trailing empty lines from the multiline string.

---

Ah, TIL. I hate Googling for odd syntax "yaml pipe dash" didn't really do a lot for me. :D